packages/standard-components/src/DataFormWide.svelte

```diff
@@ -28,14 +35,41 @@
     schema = modelDef.schema
   }
   async function save() {
+    // prevent double clicking firing multiple requests
+    if (saving) return
+    saving = true
     const SAVE_RECORD_URL = `/api/${model}/records`
     const response = await _bb.api.post(SAVE_RECORD_URL, newModel)
     const json = await response.json()
-    store.update(state => {
-      state[model] = state[model] ? [...state[model], json] : [json]
-      return state
-    })
+
+    if (response.status === 200) {
+      store.update(state => {
+        state[model] = state[model] ? [...state[model], json] : [json]
+        return state
+      })
+      
+      resetForm()
+
+      saved = true
+      setTimeout(() => {
```

Comment:
Since this is used across 2 forms and will probably be used in other components, it might be worth abstracting it out into a `debounce` function somewhere, or just pulling in `lodash.debounce` to the `standard-components`

Reply:
i dont think debounce helps us here.. the purpose of this is to make the notifier appear, then disappear after 1 second - not to group multiples of these into one

Reply:
Ah yes - I see. All good 👍 Was thinking of the `saving` boolean above.